packages/components/pages/f-checkout/src/tenants/en-GB.js

```diff
@@ -151,6 +151,24 @@ const messages = {
                 buttonText: 'Back to order'
             },
 
+            KITCHEN_NOTE_NOT_ACCEPTED: {
```

Comment:
Just checking, do these error messages mean that there was a problem submitting the note or were the customer's notes actually rejected?

Reply:
They were rejected because the config changed as they were checking out. It's a miniscule edge case, but this was the agreed copy from product.